coderd/webpush.go

```diff
@@ -0,0 +1,160 @@
+package coderd
+
+import (
+	"database/sql"
+	"errors"
+	"net/http"
+	"slices"
+
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/coderd/database/dbtime"
+	"github.com/coder/coder/v2/coderd/httpapi"
+	"github.com/coder/coder/v2/coderd/httpmw"
+	"github.com/coder/coder/v2/coderd/rbac"
+	"github.com/coder/coder/v2/coderd/rbac/policy"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+// @Summary Create user webpush subscription
+// @ID create-user-webpush-subscription
+// @Security CoderSessionToken
+// @Accept json
+// @Tags Notifications
```

Comment:
Tags: WebPush ?

Reply:
Turns out that `apidocsgen` will happily create a new empty markdown file with no content in this case. Going to leave them under notifications for now.